GameData/Data.cs

```diff
@@ -20,12 +21,17 @@ public List<Character> CreateAllGameCharacters()
 
         public string GetGameIntroduction()
         {
-            return "Once, there were three ancient instruments." + 
-                "The Harp, Piano, and the Drum." + 
-                "Inside of each instrument there was a magical gem." +
-                "A ruby, sapphire, and a diamond." +
-                "When the gems are merged, it will the create an ancient weapon." +
-                "The Moonrake."; 
+            return "Once, there were three ancient instruments." +
+                " The Harp, Piano, and the Drum." +
+                " Inside of each instrument there was a magical gem." +
+                " A ruby, sapphire, and a diamond." +
+                " When the gems are merged, it will the create an ancient weapon;" +
+                " The Moonrake." +
+                "                                                                                                                            Hello, wellcome to Moonrake, a text adventure game. " + 
+                " If you want to take a look at settings, type in settings, and if you want to save, type save." +
+                "";
```

Comment:
If there should be an extra line return after the introduction then let's put that in the engine so all games get it.